util/settings/settings.go

```diff
@@ -305,12 +305,7 @@ func (mgr *SettingsManager) GetKustomizeBuildOptions() (string, error) {
 	if err != nil {
 		return "", err
 	}
-	value, ok := argoCDCM.Data[kustomizeBuildOptions]
-	if !ok {
-		return "", err
-	}
-
-	return value, nil
+	return argoCDCM.Data[kustomizeBuildOptions], nil
```

Comment:
This is fair since we do not want to return an error if `kustomizeBuildOptions` has not been set 👍